code/gui/griddisplay.cpp

```diff
@@ -52,18 +68,62 @@ static void swap_rect_coords(int &x1, int &y1, int &x2, int &y2) {
     }
 }
 
+int GridDisplay::default_weight() {
+    return DEFAULT_WEIGHT;
+}
+
+int GridDisplay::grid_size() {
+    return GRID_SIZE;
+}
+
+double GridDisplay::scene_width() {
+    return m_scene->width();
```

Comment:
this function isn't called anywhere

Reply:
Yeah I thought I was gonna need this stuff but it turned out it wasn't needed.